server/proxy/uninstall.go

```diff
@@ -4,20 +4,28 @@
 package proxy
 
 import (
-	"fmt"
-
+	"github.com/nicksnyder/go-i18n/v2/i18n"
```

Comment:
1/5 that it's cleaner to use only use `github.com/mattermost/mattermost-plugin-api/i18n` and drop the dependency on `github.com/nicksnyder/go-i18n/v2/i18n`.

Reply:
In order for the automation to work, we need to use the `i18n.LocalizeConfig` structure from the `nicksnyder` package, so as far as I know, we cannot get rid of this dependency.

Reply:
The plugin api package also exports these: https://pkg.go.dev/github.com/mattermost/mattermost-plugin-api@v0.0.19/i18n#LocalizeConfig

Reply:
Not sure why, but using those make the extraction tool not to find the strings. My guess is that the reflection mechanism thinks that they are different types.